Tools/WebKitTestRunner/InjectedBundle/TestRunner.cpp

```diff
@@ -274,6 +274,14 @@ void TestRunner::forceImmediateCompletion()
     if (!injectedBundle.isTestRunning())
         return;
 
+    ALLOW_DEPRECATED_DECLARATIONS_BEGIN
+    bool mainFrameIsRemote = WKBundleFrameIsRemote(WKBundlePageGetMainFrame(injectedBundle.pageRef()));
```

Comment:
We don't want to increase use of WKBundleFrameIsRemote.  We should instead be moving state to the UI process and sending a sync message, the reply to which indicates if we should dump now.  It probably needs to be tied with TestRunner::notifyDone.